include/m_npc_walk.h

```diff
@@ -7,102 +7,103 @@
 #define NPCW_GET_WALK_NUM(x) (int)((x) / 3)
 #define NPCW_MAX NPCW_GET_WALK_NUM(ANIMAL_NUM_MAX)
 
-typedef struct mNpcW_c {
+typedef struct NpcWalk {
     /* 0x00 */ char unk00[0x7C];
-} mNpcW_c; // size = 0x7C
+} NpcWalk; // size = 0x7C
 
-typedef struct mNpcW_GoalData_c {
+typedef struct NpcWalkGoalData {
     /* 0x0 */ u8* types;
     /* 0x1 */ u8 count;
     /* 0x2 */ s32 time;
-} mNpcW_GoalData_c; // size = 0x6
+} NpcWalkGoalData; // size = 0x6
 
-typedef struct mNpcW_GoalDataTable_c {
-    /* 0x0 */ mNpcW_GoalData_c* data;
+typedef struct NpcWalkGoalDataTable {
+    /* 0x0 */ NpcWalkGoalData* data;
     /* 0x4 */ int count;
-} mNpcW_GoalDataTable_c; // size = 0x8
+} NpcWalkGoalDataTable; // size = 0x8
 
-typedef struct mNpcW_Appear_c {
+typedef struct NpcWalkAppear {
     /* 0x0 */ u8 status;
     /* 0x1 */ u8 way;
-} mNpcW_Appear_c; // size = 0x2;
+} NpcWalkAppear; // size = 0x2;
 
-typedef struct mNpcW_info_c {
+typedef struct NpcWalkInfo {
     /* 0x00 */ AnmPersonalID_c id;
     /* 0x0C */ s32 idx;
     /* 0x10 */ u8 status;
     /* 0x11 */ u8 type;
     /* 0x12 */ u8 goalX;
     /* 0x13 */ u8 goalZ;
-    /* 0x14 */ mNpcW_Appear_c appearInfo;
-} mNpcW_info_c; // size = 0x18
+    /* 0x14 */ NpcWalkAppear appearInfo;
+} NpcWalkInfo; // size = 0x18
 
-typedef struct mNpcW_walk_c {
-    /* 0x00 */ mNpcW_info_c info[NPCW_MAX];
+typedef struct NpcWalking {
+    /* 0x00 */ NpcWalkInfo info[NPCW_MAX];
     /* 0x78 */ u16 idxUse;
     /* 0x7A */ u8 infoUse;
     /* 0x7B */ u8 infoMax;
-} mNpcW_walk_c; //size = 0x7C
+} NpcWalking; //size = 0x7C
 
-enum {
+typedef enum WalkGoal{
     /* 0 */ NPCW_GOAL_SHRINE,    /* Shrine/Wishing Well acre */
     /* 1 */ NPCW_GOAL_HOME,      /* Randomly selected acre with a villager house */
     /* 2 */ NPCW_GOAL_ALONE,     /* Randomly selected acre with no other animals in it */
     /* 3 */ NPCW_GOAL_MY_HOME,   /* Acre containing the villager's house */
 
     /* 4 */ NPCW_GOAL_NUM
```

Comment:
We usually use `MAX` instead of `NUM` for those, and no empty line between the other entries and this one, but since there are other enums in the repo like this I guess it is fine to leave them as they are for now, we could clean them up in the future